Robinhood/Robinhood.py

```diff
@@ -757,3 +757,79 @@ def place_sell_order(
         """
         transaction = Transaction.SELL
         return self.place_order(instrument, quantity, bid_price, transaction)
+    
+    ##############################
+    #GET OPEN ORDER(S)
+    ##############################
+    
+    def get_open_orders(self):
+        """
+        Returns all currently open (cancellable) orders.
+        If not orders are currently open, `None` is returned.
+        TODO: Is there a way to get these from the API endpoint without stepping through order history?
+        """
+
+        open_orders = []
+        orders = self.order_history()
+        for order in orders['results']:
+            if(order['cancel'] is not None): 
+                open_orders.append(order)
+
+        if len(open_orders) > 0:
+            return open_orders
+
+    ##############################
+    #CANCEL ORDER(S)
+    ##############################
+
+    def cancel_order(
+            self,
+            orderID
+    ): 
+        """
+        Cancels specified order and returns the response (results from `orders` command). 
+        If order cannot be cancelled, `None` is returned.
+
+        Args:
+            orderID (str): Order ID that is to be cancelled
+
+        """
+        order = self.session.get(self.endpoints['orders'] + orderID).json()
+
+        if order['cancel'] is not None:
+            try: 
+                req = self.session.post(order['cancel'])
+                req.raise_for_status()
+            except requests.exceptions.HTTPError as err_msg:
+                warnings.warn('Failed to cancel order ID: '+ order['id'] + repr(err_msg))
+                return None
+        
+        else: #no cancel link, cannot cancel
+            warnings.warn('Unable to cancel order ID: '+ order['id'])
+            return None
+
+        return order
+
+    def cancel_open_orders(self):
+        """
+        Cancels all open orders and returns the responses from the canelled orders (results from `orders` command). 
+        If no orders are open or no orders were cancelled (i.e. failed to cancel), returns `None`
+        """
+        cancelled_orders = []
+        open_orders = self.get_open_orders()
+
+        if open_orders is None: #no open orders
+            return None
+
+        for order in open_orders:
+            if(order['cancel'] is not None): 
+                try: 
+                    req = self.session.post(order['cancel'])
+                    req.raise_for_status()
+                    cancelled_orders.append(order)
+
+                except requests.exceptions.HTTPError as err_msg:
+                    warnings.warn('Failed to cancel order ID: '+ order['id'] + repr(err_msg))
```

Comment:
This mangles the http error message with the order id so it looks like `Failed to cancel order ID: 4HttpError(id=404, msg='page not found')`

Would suggest adding a space.